uber/tests/models/test_badge_funcs.py

```diff
@@ -172,6 +172,26 @@ def test_self_assignment(self, session):
         change_badge(session, session.staff_five,  c.STAFF_BADGE, new_num=5)
 
 
+class TestNonShiftChecks:
+    @pytest.fixture(autouse=True)
+    def set_shift_badges_off(self, monkeypatch):
+        monkeypatch.setattr(c, 'SHIFT_CUSTOM_BADGES', False)
+
+    @pytest.fixture(autouse=True)
+    def teardown_range_check(request):
+        return True  # TODO: make this actually check ranges
```

Comment:
It's fine to just literally say ``if not c.SHIST_CUSTOM_BADGES:`` in the ``teardown_range_check`` function.  Part of the point of the global ``c`` object is to make unit testing easier by allowing us to have a canonical place to look for this stuff that can also be monkeypatched.